qiskit_dynamics/type_utils.py

```diff
@@ -320,39 +322,125 @@ def vec_dissipator(L: Array):
     )
 
 
-def to_array(op: Union[Operator, Array, List[Operator], List[Array], spmatrix]):
-    """Convert an operator or list of operators to an Array.
+def isinstance_qutip_qobj(obj):
```

Comment:
Don't forget type hints here. You can use: `any` to refer to `obj` being able to literally be anything. Also the doc string needs proper:
```
Args:
     obj: Object to check.

Returns:
 ....
```